atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/KeyValueService.java

```diff
@@ -700,4 +712,11 @@ default boolean isInitialized() {
     default boolean performanceIsSensitiveToTombstones() {
         return false;
     }
+
+    /**
+     * @return If {@link #compactInternally(TableReference)} should be called to free disk space.
+     */
+    default boolean shouldManuallyCompact() {
```

Comment:
The naming of this is a bit strange, though I'm not sure I can think of something much better...

Reply:
Is it a bit weird - we're saying that Atlas is manually compacting... `shouldCompactAfterSweep`?

Reply:
Discussed offline - I think we have `shouldTriggerCompactions` now which I like